localstack/aws/scaffold.py

```diff
@@ -343,15 +343,26 @@ def generate_service_api(output, service: ServiceModel, doc=True):
         output.write("        raise NotImplementedError\n")
 
 
-@click.command()
+@click.group(name="scaffold", invoke_without_command=True)
+@click.pass_context
+def scaffold(ctx):
+    # FIXME: doesn't invoke subcommand properly
```

Comment:
not sure what this fixme is about. i think this is the pattern for this type of situation. maybe i'm missing something?

Reply:
While this would be the pattern for a default command, the problem is that the service name for the `generate` command is an argument and not an option, so a call to the generate command without explicitly stating the command would look as follows:
```
python3 -m localstack.aws.scaffold opensearch
```

Unfortunately, this is ambiguous, because the service name argument would be misinterpreted as the command.
Imho, I think it would also be a bit confusing and there's no need for a stable API / migration path on the scaffold yet, so I'll just remove the default command functionality for now.